tests/LogEnhancerTest.php

```diff
@@ -13,20 +12,18 @@ class LogEnhancerTest extends TestCase
     /** @test */
     public function it_adds_request_details_to_logs()
```

Comment:
Add two tests. One by enabling all config values. Second by disabling all of them.